cfme/tests/openstack/cloud/test_instances.py

```diff
@@ -176,12 +176,12 @@ def test_list_vms_infra_node(appliance, provider, soft_assert):
     host_collection = appliance.collections.hosts
     # Match hypervisors by IP with count of running VMs
     hvisors = {hv.host_ip: hv.running_vms for hv in provider.mgmt.api.hypervisors.list()}
+    hv_ips = hvisors.keys()
 
     # Skip non-compute nodes
     hosts = [host for host in host_collection.all(provider) if 'Compute' in host.name]
     assert hosts
-    for host in hosts:
-        host_ip = host.get_detail('Properties', 'IP Address')
+    for c, host in enumerate(hosts):
```

Comment:
REQUIRED: please don't use variables with name like "c". This significantly decreases readability.  Variables should have self explaining names.